lost/src/test/java/com/mapzen/android/lost/internal/FusedLocationProviderApiImplTest.java

```diff
@@ -219,7 +219,7 @@ public void setMockMode_shouldNotRegisterDuplicateListeners() throws Exception {
         api.requestLocationUpdates(request, listener);
         api.setMockMode(false);
         api.requestLocationUpdates(request, listener);
-        assertThat(shadowLocationManager.getRequestLocationUpdateListeners()).hasSize(2);
+        assertThat(api.getListeners()).hasSize(2);
```

Comment:
Similarly this test was added to ensure we were not registering duplicate listeners with the `LocationManager` when exiting mock mode (which was an actual bug at some point)